Jenkinsfile

```diff
@@ -728,7 +728,11 @@ pipeline {
                 stage('win-RS1') {
                     when {
                         beforeAgent true
-                        expression { params.windowsRS1 }
+                        // Skip this stage on PRs unless the windowsRS1 checkbox is selected
+                        anyOf {
```

Comment:
can you make this same change to the other stages we disable on PR's?

Reply:
I'm not sure that would make sense for the existing `s390x` and `s390x integration-cli` stages without a new build parameter. Could be an idea for a future PR.

For this PR, I'd prefer to keep the scope to reducing CI times by disabling windows RS1.

Reply:
oh 🤦‍♂ you're right, I thought we had a parameter for those, but `ppc64le` is for the stage that's enabled by default